app/javascript/react/components/Navbar/Navbar.tsx

```diff
@@ -30,68 +31,91 @@ const NavItem = ({
       </a>
       <S.Container>
-        <S.BuyCTA href={urls.airbeamBuy}>get airbeam</S.BuyCTA>
+        <S.BuyCTA href={urls.airbeamBuy}>
+          {t("navbar.sections.getAirBeam")}
+        </S.BuyCTA>
         <nav>
           <S.MenuButton onClick={() => setNavMenuVisible(true)}>
-            <img src={hamburger} alt="Menu icon" aria-label="Open menu"></img>
+            <img
+              src={hamburger}
+              alt={t("navbar.altMenu")}
+              aria-label="Open menu"
+            ></img>
           </S.MenuButton>
         </nav>
       </S.Container>
       <S.NavList $isVisible={navMenuVisible}>
         <S.NavHeader>
           <a href={urls.habitatMap} aria-label="HabitatMap page">
-            <img src={habitatMapLogo} alt="Habitatmap logo" />
+            <img src={habitatMapLogo} alt={t("navbar.altHabitatLogo")} />
           </a>
           <S.Button onClick={() => setNavMenuVisible(false)}>
             <img
               src={iconNavClose}
-              aria-label="Close navigation menu"
-              alt="Close icon"
+              aria-label={t("navbar.altClose")}
+              alt={t("navbar.altClose")}
             />
           </S.Button>
         </S.NavHeader>
         <NavItem isNavTitle href={urls.airbeam}>
-          AirBeam
+          {t("navbar.sections.airBeam")}
         </NavItem>
         <S.SubNav>
-          <NavItem href={urls.airbeamUserStories}>User Stories</NavItem>
-          <NavItem href={urls.airbeamHowItWorks}>How it Works</NavItem>
-          <NavItem href={urls.airbeamFaq}>FAQ</NavItem>
-          <NavItem href={urls.airbeamUsersGuide}>User's Guide</NavItem>
-          <NavItem href={urls.airbeamBuy}>Buy it Now</NavItem>
+          <NavItem href={urls.airbeamUserStories}>
+            {t("navbar.sections.userStories")}
+          </NavItem>
+          <NavItem href={urls.airbeamHowItWorks}>
+            {t("navbar.sections.work")}
+          </NavItem>
+          <NavItem href={urls.airbeamFaq}>{t("navbar.sections.faq")}</NavItem>
+          <NavItem href={urls.airbeamUsersGuide}>
+            {t("navbar.sections.usersGuide")}
+          </NavItem>
+          <NavItem href={urls.airbeamBuy}>
+            {t("navbar.sections.buyNow")}
+          </NavItem>
         </S.SubNav>
         <NavItem isNavTitle href={urls.aircasting}>
-          AirCasting
+          {t("navbar.sections.airCasting")}
         </NavItem>
         <S.SubNav>
-          <NavItem href={urls.map}>AirCasting Maps</NavItem>
-          <NavItem href={urls.android}>Android App</NavItem>
-          <NavItem href={urls.iOS}>iOS App</NavItem>
-          <NavItem href={urls.actions}>AirCasting Actions</NavItem>
+          <NavItem href={urls.map}>{t("navbar.sections.maps")}</NavItem>
+          <NavItem href={urls.android}>
+            {t("navbar.sections.androidApp")}
+          </NavItem>
+          <NavItem href={urls.iOS}>{t("navbar.sections.iosApp")}</NavItem>
```

Comment:
Maybe we can adjust, so the names are corresponding? Here I like `iOSApp` better.

Reply:
Oh sorry, that's my bad. 🤦 I was mistaken with the capital letters `iosApp` both in transaltion and url. 😮‍💨